sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsError.cs

```diff
@@ -30,5 +68,55 @@ internal TextAnalyticsError(string code, string message, string target = null)
         /// the property in error).
         /// </summary>
         public string Target { get; }
+
+        /// <summary> Generated Error code. </summary>
+        internal ErrorCodeValue Code { get; }
+        /// <summary> Generated inner error contains more specific information. </summary>
+        internal InnerError Innererror { get; }
+        /// <summary> Generated details about specific errors that led to this reported error. </summary>
+        internal IReadOnlyList<TextAnalyticsError> Details { get; }
```

Comment:
We don't use these 3, so hidden them from users. We still need to initialize them in constructor